CMakeLists.txt

```diff
@@ -200,6 +198,7 @@ set(PROJECT_SOURCE_LIST_C
 )
 
 if( WIN32 )
+    set(PROJECT_SOURCE_LIST_C ${PROJECT_SOURCE_LIST_C} pcap-win32.c )
```

Comment:
That's what the

    set(PROJECT_SOURCE_LIST_C ${PROJECT_SOURCE_LIST_C} pcap-${PCAP_TYPE}.c)

does after the code to determine PCAP_TYPE.  Why is this necessary?

Reply:
See comment added to the pull request itself. pcap-win32.c would contain the DllMain entry point, and so should then be compiled systematically on Windows, whereas capture driver implementations would be compiled depending on which Cmake option is enabled or which library was detected.